arangodb-net-standard/CursorApi/ICursorApiClient.cs

```diff
@@ -35,11 +40,16 @@ public interface ICursorApiClient
                 long? memoryLimit = null,
                 int? ttl = null,
                 string transactionId = null,
+                CursorHeaderProperties headerProperties = null,
             CancellationToken token = default);
 
         /// <summary>
         /// Execute an AQL query, creating a cursor which can be used to page query results.
         /// </summary>
+        /// <remarks>
+        /// This method supports Read from Followers (dirty-reads). 
+        /// To enable it, set the AllowReadFromFollowers header property to true.
```

Comment:
Maybe use an XML tag reference here? Would help if it's renamed in future.

Example: `set the <see cref="ApiHeaderProperties.AllowReadFromFollowers"/> header property to true.`

Reply:
Done